frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsPersistent.tsx

```diff
@@ -7,57 +7,81 @@ import {
   PlayCircleOutlined,
 } from '@ant-design/icons';
 import { Instance } from '../../../../utils';
+import { useApplyInstanceMutation } from '../../../../generated-types';
+import { setInstanceRunning } from '../../ActiveUtils';
 
 export interface IRowInstanceActionsPersistentProps {
   extended: boolean;
   instance: Instance;
-  startInstance: (idInstance: string, idTemplate: string) => void;
-  stopInstance: (idInstance: string, idTemplate: string) => void;
 }
 
 const RowInstanceActionsPersistent: FC<IRowInstanceActionsPersistentProps> = ({
   ...props
 }) => {
-  const { extended, instance, startInstance, stopInstance } = props;
+  const { extended, instance } = props;
 
-  const { status, idTemplate, name } = instance;
+  const { status, running } = instance;
 
   const font22px = { fontSize: '22px' };
 
+  const [disabled, setDisabled] = useState(false);
+
+  const [applyInstanceMutation] = useApplyInstanceMutation();
+
+  const setInstanceStatus = async () => {
+    if (!disabled) {
+      setDisabled(true);
+      try {
+        const result = await setInstanceRunning(
+          running!,
+          instance,
+          applyInstanceMutation
+        );
+        if (result) setTimeout(() => setDisabled(false), 400);
+      } catch {
+        //do nothing at the moment
+      }
+    }
+  };
+
   return status === 'VmiReady' ? (
     <Tooltip placement="top" title={'Pause'}>
       <Button
+        loading={disabled}
         className={`hidden ${
           extended ? 'sm:block' : 'xs:block'
         } flex items-center`}
         type="warning"
         with="link"
         shape="circle"
         size="middle"
+        disabled={disabled}
         icon={
           <PauseCircleOutlined
             className={'flex justify-center items-center'}
             style={font22px}
           />
         }
-        onClick={() => startInstance(name, idTemplate!)}
+        onClick={() => setInstanceStatus()}
```

Comment:
```suggestion
        onClick={setInstanceStatus}
```
also below, don't create new lambdas when it's not necessary